packages/core/src/components/MenuButton/MenuButton.tsx

```diff
@@ -366,10 +361,7 @@ const MenuButton: VibeComponent<MenuButtonProps> & {
       </Tooltip>
     );
 
-    if (showTooltipOnlyOnTriggerElement) {
-      return dialogNode(tooltipNode(triggerElementNode));
-    }
-    return tooltipNode(dialogNode(triggerElementNode));
+    return dialogNode(tooltipNode(triggerElementNode));
```

Comment:
you can inline it I guess and remove the consts, if I remember correctly I've only added them for `showTooltipOnlyOnTriggerElement` functionality